system-tests/resources/appconfig/consumer-eu/app.config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "apiKey": "ApiKeyDefaultValue",
+  "catalogUrl": "http://localhost:8182/api/federatedcatalog",
+  "dataManagementApiUrl": "http://localhost:9192/api/v1/data",
+  "storageAccount": "consumereuassets",
+  "storageExplorerLinkTemplate": "storageexplorer://v=1",
```

Comment:
nit: will this link works as expected? I mean when we click this in UI, will it open and highlight the correct transferred asset.

in terraform we have something like
storageexplorer://v=1&accountid=${azurerm_resource_group.participant.id}/providers/Microsoft.Storage/storageAccounts/{{account}}&subscriptionid=${data.azurerm_subscription.current_subscription.subscription_id}&resourcetype=Azure.BlobContainer&resourcename={{container}}


Just a question,

Reply:
Direct links to Storage Explorer are only supported for subscription nodes, not for local accounts.

This link provides at least that the Storage Explorer opens (I've tested this). Otherwise, we would have a dead button in the MVD UI.

For reference:
https://docs.microsoft.com/en-us/azure/storage/common/storage-explorer-direct-link#direct-link-limitations

> Direct links are only supported for resources under subscription nodes. Additionally, only the following resource types are supported:

> Storage Accounts
Blob Containers
Queues
File Shares
Tables

Reply:
I would mention in the doc that the "Open in Storage Explorer" button does not work, can be confusing otherwise